nullaway/src/test/java/com/uber/nullaway/tools/SerializationTestHelper.java

```diff
@@ -24,125 +24,139 @@
 
 import static org.junit.Assert.fail;
 
+import com.google.common.base.Preconditions;
 import com.google.errorprone.CompilationTestHelper;
 import com.uber.nullaway.NullAway;
-import com.uber.nullaway.fixserialization.out.SuggestedFixInfo;
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.nio.charset.Charset;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class SerializationTestHelper {
-  private FixDisplay[] expectedOutputFixes;
-  private final Path outputPath;
+public class SerializationTestHelper<T> {
```

Comment:
```suggestion
public class SerializationTestHelper<T extends Display> {
```
See other comment about `Display` interface

Reply:
772011ddeb38cd55871cbbc06abb0ffbfa0e5879